sec2_the_ruby_programming_language/2.3_number.rb

```diff
@@ -0,0 +1,75 @@
+# Numbers
+puts 1 + 2       #3
+puts 2 - 1       #1
+puts 10 * 2      #20
+puts 10 / 4      #2
+puts 10.0 / 4    #2.5
+puts 10 / 4.0    #2.5
+puts 10 / 4.to_f #2.5
+puts (10/4).to_f #2.0
+
+# Assign variables
+x = 5
+y = 10
+puts y / x #2
```

Comment:
Based on line 1-9, this formula will yield an incorrect result no?

Reply:
Is the incorrect result referring to the conversion?

Reply:
wat do you mean?

Reply:
This will not yield an incorrect result.

Reply:
Change x to 3. Do you get 3.33333 or 3?
The correct answer should be 3.333333333 right?

Reply:
The answer will remain as 3 unless it is converted to become a float number.

Reply:
Hello. Please keep up with the conversation.
Please read back on the **entire** conversation and if don't understand what we are discussing, please ask

Reply:
So the question here is asking if we assign `3` to `x`, after `10/3`, why the answer will become `3` instead of `3.3333`. This is due to the reason mentioned [above](https://github.com/ziqiang2344/Learn_Ruby_on_Rails/pull/4#discussion_r851601181).